### PR TITLE
Fixed LeNet architecture not working

### PR DESCRIPTION
**Pull request by mmasana**
LeNet seemed to not be working properly, I think this is a fix.

@btwardow : can you take a look? maybe this is also what was affecting the tests

### CODE REVIEW

File: src/networks/lenet.py [hunk 1]
```diff
@@ -28,3 +28,9 @@ def forward(self, x):
         out = F.relu(self.fc2(out))
         out = self.fc(out)
         return out
+
+
+def lenet(pretrained=False, **kwargs):
```
Comment (btwardow) [inline src/networks/lenet.py]:
```suggestion
def LeNet(pretrained=False, **kwargs):
```

File: src/networks/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from torchvision import models
 
-from .lenet import LeNet
+from .lenet import lenet
```
Comment (btwardow) [inline src/networks/__init__.py]:
```suggestion
from .lenet import LeNet
```

**Review by btwardow — CHANGES_REQUESTED** [review]
Hi Marc! I think that's a good catch :wink: However, I'd leave `LeNet` as a base name and do not change to `lenet`, so then we don't have to modify anything else as the naming is consistent, i.e. in the `network/__init__.py` or test files.

After the change, I run tests. They are still running, but all end being green :clap:

**Comment by mmasana** [comment]
Makes sense to leave the old nomenclature. I had not thought of it so much as just trying to fix it. Nice!

**Pull request merged**